<launch>
    <param name="names" textfile="$(find vision)/scripts/class-names.txt" />
    <arg name="model_path" default="$(find vision)/imports/vision/MSCV2/yv3m-256-testing.pt" />
    <arg name="model_config_path" default="$(find vision)/imports/vision/MSCV2/cfg/yolov3-mid-256.cfg"/>
    <param name="model_path" value="$(arg model_path)"/>
    <param name="model_config_path" value="$(arg model_config_path)" />
    <node name="vision_subscriber" pkg="vision" type="VisionSubscriber.py" />
</launch>